cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,58 @@
+import PageObject from '../PageObject';
+
+export class SettingsPageObject extends PageObject {
+    url = '/settings';
+
+    get usernameField() {
+       return cy.getByDataCy('update-username');
+    }
+
+    typeUsername(username) {
+        this.usernameField.clear().type(username);
+      }
+
+    assertProfilePage(username) {
+        return cy.url().should('contain', `/profile/${username}`);
+      }
+
+      get submitBtn() {
+        return cy.getByDataCy('submit-btn');
+      }
+
+      submitForm() {
+        this.submitBtn.click();
+      }
+
+    get bioField() {
+        return cy.getByDataCy('update-bio');
+      }
+
+      typeBio(bio) {
+        this.bioField.type(bio);
+      }
+
+      get emailField() {
+        return cy.getByDataCy('update-email');
+      }
+
+      typeEmail(email) {
+        this.emailField.clear().type(email);
+      }
+
+      get passwordField() {
+        return cy.getByDataCy('update-password');
+      }
+
+      typePassword(password) {
+        this.passwordField.type(password);
```

Comment:
Consider clearing the password field before typing to ensure that any existing text is removed. This will make the behavior consistent with the typeUsername and typeEmail methods.